src/components/BB-BUS/AZFilterElement.js

```diff
@@ -32,4 +32,10 @@ export const AZFilterElement = styled.View`
     css`
       margin-right: ${normalize(14)}px;
     `};
+
+  ${(props) =>
+    props.blackBackgroundForContrast &&
```

Comment:
please don't call it `black`, as this can be different in some cases. `backgroundForContrast` should be enough to tell what is going on.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/c8b47067d02ff0a1dc6b6832058238818b26ea70